src/engine/net/Server.hpp

```diff
@@ -43,27 +47,24 @@ class Server {
     virtual void OnMessage(const std::shared_ptr<Connection> &client,
                            Message &message) = 0;
 
-public:
-    // Called when a client is validated
-    virtual void
-    OnClientValidated(const std::shared_ptr<Connection> &client) = 0;
+    void ClearDisconnectedClients();
 
+public:
 private:
     void HandleAcceptedConnection(std::error_code ec,
                                   asio::ip::tcp::socket socket);
 
-public:
-    MessageQueue<OwnedMessage> m_incomingMessages;
-    std::vector<std::shared_ptr<Connection>> m_connections;
-
-private:
     asio::io_context m_ioContext;
     std::thread m_contextThread;
 
     asio::ip::tcp::acceptor m_acceptor;
 
     // Start ID
     uint32_t m_idCounter = 1;
+
+public:
+    MessageQueue<OwnedMessage> m_incomingMessages;
+    std::vector<std::shared_ptr<Connection>> m_connections;
```

Comment:
should these really be public? In that case remove the `m_` prefix, otherwise make them protected/private with a public getter.

Reply:
They were already public, I just moved them down to avoid having 
```
public:
private:
public:
private:
```

Reply:
yes, although it should be fixed anyways.